plugins/metadata/services/MetadataProfileService.php

```diff
@@ -439,8 +439,6 @@ function updateDefinitionFromFileAction($id, $xsdFile)
 			throw new KalturaAPIException(MetadataErrors::METADATA_UNABLE_TO_TRANSFORM, $e->getMessage());
 		}
 		
-		$dbMetadataProfile->save();
```

Comment:
Keep the save, you should save in the same method that you call the setters.
Calling save without any change won't trigger a db transaction so I prefer to many saves than not enough.